### ISSUE TITLE
Missing descriptions in Vega-Lite schema.

### ISSUE THREAD

**Issue by jheer** [vega/vega-lite#7330]
Consider this page of automatically generated documentation for the VL API:
https://vega.github.io/vega-lite-api/api/radio

Not very useful, due to missing descriptions in the VL schema. I'm guessing this is due to pass through from the Vega schema. Can you verify this? If so, would it be sufficient to add description fields to the Vega schema? In that case, I can open a new issue with Vega, perform an audit, and fix this issue there.

(In any case let's keep this issue here for tracking purposes -- please don't transfer it to vega/vega.)

**Comment by domoritz** [comment]
Yes, we import Binding directly from Vega in https://github.com/vega/vega-lite/blob/e8e45e05342428e3b7dbb83e23e828fe9dee7c1a/src/parameter.ts#L27.

**Comment by domoritz** [comment]
I misread your message. We use the comments in the typings, not the schema!
